lib/modules/manager/heroku-runtime/extract.ts

```diff
@@ -0,0 +1,12 @@
+import { DockerDatasource } from '../../datasource/docker';
+import type { PackageDependency, PackageFileContent } from '../types';
+
+export function extractPackageFile(content: string): PackageFileContent {
+  const dep: PackageDependency = {
+    depName: 'python',
+    commitMessageTopic: 'Python',
+    currentValue: content.replaceAll('python-', '').trim(),
```

Comment:
is there only one line in this file? can there be other values? we should validate and document that only python is yet supported.

maybe also add a preview state note.

Reply:
changed to use a regex match group